src/c5t.js

```diff
@@ -123,137 +257,400 @@
     xhr.send(payloadString);
   };
   
-  _transports[str_image] = function (baseUrl, payloadString, callbackFn) {
+  /**
+   * A `GET` transport via `Image`.
+   */
+  _transports[STR_image] = function (baseUrl, payloadString, callbackFn) {
     var image = new Image();
     image.onload = image.onerror = function () {
       image.onload = image.onerror = null;
       image = null;
       callbackFn();
     };
-    image.src = baseUrl + '?' + payloadString;
+    image.src = baseUrl + "?" + payloadString;
   };
   
   // If the `sendBeacon` API is not supported we don't add the transport implementation.
   if (navigator.sendBeacon) {
-    _transports[str_beacon] = function (baseUrl, payloadString, callbackFn) {
+    /**
+     * A `POST` transport via `sendBeacon`.
+     */
+    _transports[STR_beacon] = function (baseUrl, payloadString, callbackFn) {
       navigator.sendBeacon(baseUrl, payloadString);
       setTimeout(callbackFn, 0);
     };
   }
   
+  function _serializeValue(value) {
+    return (
+      value === true ? "1" :
+      value === false ? "0" :
+      void 0 == value ? "" :
```

Comment:
`void 0` returns the true `undefined`; `undefined == value` evaluates to true if `value` is `undefined` or `null`.
Honestly, I've copied this expression from GA.